hpeOneView/resources/security/users.py

```diff
@@ -108,8 +108,148 @@ def get_by_userName(self, userName):
             uri = self.URI + '/role/' + userName
             self._helper.validate_resource_uri(uri)
             data = self._helper.do_get(uri)
-            new_resource = self.new(self._connection, data)
+            result = data['members']
+            return result
         else:
-            new_resource = None
+            return None
+
+    def get_by_userName(self, name):
+        """
+        Gets a complete json body for username
+
+        Args:
+          name: userName of the user
+
+        Returns:
+           dict: User
+        """
+
+        uri = self._helper.build_uri(name)
+        new_resource = self.get_by_uri(uri)
```

Comment:
use ```return super(Users, self).get_byr_uri(uri)``` instead of assigning return value to a variable and returning that variable.